File: backend/src/main/java/com/codecool/training_portal/dto/group/project/questionnaire/QuestionnaireSubmissionResponseDto.java
package com.codecool.training_portal.dto.group.project.questionnaire;

import java.util.List;

public record QuestionnaireSubmissionResponseDto(
  Long id, String name, String description, List<SubmittedQuestionResponseDto> questions,
  Integer receivedPoints, Integer maxPoints, String createdAt) {
}

File: frontend/src/questionnaires/dto/QuestionnaireSubmissionResponseDto.ts
import {SubmittedQuestionResponseDto} from "./SubmittedQuestionResponseDto.ts";

export interface QuestionnaireSubmissionResponseDto {
  id: number;
  name: string;
  description: string;
  questions: SubmittedQuestionResponseDto[];
  receivedPoints: number;
  maxPoints: number;
  createdAt: string;
}
